src/Services/ConsignmentEncode.php

```diff
@@ -245,7 +245,7 @@ private function encodeCdCountry()
                     'contents' => 1,
                     'weight'   => $consignment->getTotalWeight(),
                     'items'    => $items,
-                    'invoice'  => $consignment->getInvoice(),
+                    'invoice'  => ! empty($consignment->getInvoice()) ? $consignment->getInvoice() : '',
```

Comment:
Werkt dit niet?
```suggestion
                   $consignment->getInvoice() ?? '',
```

Reply:
Zal niet werken omdat je dan de volgende error krijgt:
`data.shipments[0].customs_declaration.invoice Boolean value found, but a string or a null is required`

Reply:
Done